etcdserver/etcdhttp/client.go

```diff
@@ -50,6 +50,7 @@ const (
 	deprecatedMachinesPrefix = "/v2/machines"
 	membersPrefix            = "/v2/members"
 	statsPrefix              = "/v2/stats"
+	statusPrefix             = "/v3/status"
```

Comment:
sorry. i should be more clear. all v3 stuff needs to be exposed via rpc not http endpoint.